src/vos/storage_estimator/common/vos_size.py

```diff
@@ -10,7 +10,7 @@
 def convert(stat):
     """Convert byte value to pretty string"""
     size = 1024 * 1024 * 1024 * 1024 * 1024
-    for mag in ['P', 'T', 'G', 'M', 'K']:
+    for mag in ['PiB', 'TiB', 'GiB', 'MiB', 'KiB', '  B']:
```

Comment:
Is `' B'` intentional? I.e. should there be an extra space?

Reply:
yeah, just looks better in the output if they all take 3 spaces.

Reply:
```
Using DAOS version: 2.3.103
  Reading VOS structures from current installation
Metadata breakdown:
	pool                :     320.00   B ( 0.01%)
	container           :    1024.00   B ( 0.02%)
	object              :     196.75 KiB ( 3.73%)
	dkey                :     308.01 KiB ( 5.84%)
	akey                :     330.67 KiB ( 6.26%)
	single_value        :       1.12 KiB ( 0.02%)
	array               :     192.19 KiB ( 3.64%)
	user_meta           :      32.03 KiB ( 0.61%)
	total_meta          :       1.01 MiB (19.51%)
Data breakdown:
	total_meta          :       1.01 MiB (19.51%)
	user_value          :       4.12 MiB (79.88%)
	total               :       5.15 MiB (100.00%)
Physical storage estimation:
	scm_total           :       1.15 MiB (22.40%)
	nvme_total          :       4.00 MiB (77.60%)
Total storage required:       5.15 MiB
```

Reply:
I see there is a bug though...Should be stat >= size